OloEngine/src/OloEngine/Renderer/Mesh.cpp

```
@@ -37,39 +88,65 @@ namespace OloEngine
     {
-        OLO_CORE_ASSERT(m_MeshSource, "MeshSource is null!");
+        if (!m_MeshSource)
+            return s_NoIndices;
+
         return m_MeshSource->GetIndices();
     }
 
     Ref<VertexArray> Mesh::GetVertexArray() const
     {
-        OLO_CORE_ASSERT(m_MeshSource, "MeshSource is null!");
+        if (!m_MeshSource)
```

Cppcheck (2.21.0)

[style] The function 'GetGeneratedCollider' is never used.

(unusedFunction)